src/Symfony/Component/Cache/Adapter/Client/MemcachedClient.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter\Client;
+
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+
+/**
+ * @author Rob Frawley 2nd <rmf@src.run>
+ */
+class MemcachedClient
```

Comment:
This class should be removed: creating a Memcached layer is not the responsibility of the Cache component. But by having this class, people *will* use it as such. An alternative could be to mark it as internal, but I'm not sure this makes more sense.